tests/Integration/PrestaShopBundle/Controller/GridControllerTestCase.php

```diff
@@ -28,9 +28,11 @@
 
 namespace Tests\Integration\PrestaShopBundle\Controller;
 
+use http\Exception\InvalidArgumentException;
```

Comment:
This namespace seems a bit weird no? Besides I don't see any use of this exception except in a comment, is it really useful?